src/ApiService/FunctionalTests/1f-api/Info.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Text.Json;
+using System.Text.Json.Nodes;
+using Xunit.Abstractions;
+
+namespace FunctionalTests {
+
+    public class InfoVersion : IFromJsonElement<InfoVersion> {
+        JsonElement _e;
+
+        public InfoVersion() { }
+        public InfoVersion(JsonElement e) => _e = e;
+        public InfoVersion Convert(JsonElement e) => new InfoVersion(e);
+
+        public string Git => _e.GetProperty("git").GetString()!;
+        public string Build => _e.GetProperty("build").GetString()!;
+        public string Version => _e.GetProperty("version").GetString()!;
+    }
+
+
+    public class InfoResponse : IFromJsonElement<InfoResponse> {
```

Comment:
Question: what is the reason we aren't reusing the service request/response types in these tests?

Reply:
Long time ago a Microsoft test manager told me that you never should use dev code to test dev code. Since that will miss bunch of bugs.

I don't think we should be doing it everywhere, but at least in one place it might be a good idea.

Reply:
Maybe it will be a good idea to use Swagger-generated definitions here, when we have them.